README.md

```diff
@@ -118,6 +118,7 @@ chmod a+x ./setup.sh
 docker-compose up -d mail
 
 ./setup.sh email add <user@domain> [<password>]
+./setup.sh alias add postmaster@<domain> <user@domain>
```

Comment:
Can we add an alias for amavis here as well, please? The system sometimes sends mail to amavis.

Reply:
I think you're right. I'd prefer doing "both", as you said, removing the entries from the README when the other PR is released. Thanks for noticing:)

Reply:
This was really meant to be added to #1709 but I forgot.  The idea is, the alias amavis->postmaster gets auto-added to /etc/aliases (also clamav->postmaster and then in the docs we prompt users to alias postmaster to a monitored inbox.

Reply:
What's your take here @erik-wramner ?